libs/dh/admin/feature-user-management/src/lib/drawer/roles/dh-role-drawer.component.ts

```diff
@@ -96,17 +96,15 @@ export class DhRoleDrawerComponent {
   private readonly store = inject(
     DhAdminUserRoleWithPermissionsManagementDataAccessApiStore
   );
```

Comment:
Nitpick: Maybe move styles to a file

Reply:
Good point. The inline styling was getting bigger, so moving it to a new file cleared the class a bit.